backend/app/repositories/models/custom_bot.py

```diff
@@ -25,6 +33,7 @@ class BotModel(BaseModel):
     owner_user_id: str
     is_pinned: bool
     embedding_params: EmbeddingParamsModel
+    generation_config: GenerationConfigModel | None
```

Comment:
To avoid bug, basically we don't want to make attributes nullable. In this case we can store default config defined on config.py if the value is empty as you implemented.

Reply:
In addition to that, could you replace the name with `generation_config_params` or `generation_params`? The reason is we'd like to align `embedding_params`